graphql/subscriptions/productEdited.graphql

```diff
@@ -0,0 +1,78 @@
+fragment ProductAttributesWebhookPayload on SelectedAttribute {
+  attribute {
+    id
+    name
+    type
+  }
+}
+
+fragment ProductWebhookPayload on Product {
+  id
+  name
+  description
+  slug
+  thumbnail {
+    url
+  }
+  attributes {
+    ...ProductAttributesWebhookPayload
+  }
+}
+
+fragment ProductVariantWebhookPayload on ProductVariant {
+  id
+  name
+  sku
+  attributes {
+    ...ProductAttributesWebhookPayload
+  }
+
+  product {
+    ...ProductWebhookPayload
+  }
+}
+
+subscription ProductEdited {
```

Comment:
I've created a single query for all of the webhooks, seemed like less work. Let me know if it's bad :P